packages/phalanx/test/Field/ViewAllocationSize.cpp

```diff
@@ -29,7 +29,7 @@ namespace phalanx_test {
 
   // ********************************************************
   template<typename ScalarT>
-  void test_span(std::ostream& out,bool success)  {
+  void test_span(std::ostream& out,bool &success)  {
```

Comment:
@rppawlo This was exposed by building with additional warnings flags. Looks like the test is actually failing:
https://sems-cdash-son.sandia.gov/cdash/tests/25826614

Reply:
Thanks @cgcgcg - I'll take a look on Monday.

Reply:
This Kokkos PR will allow the phalanx test to be reenabled. Not sure yet if this will make it into the Kokkos 5.0 release. Thanks to @crtrott for getting in a quick fix!
https://github.com/kokkos/kokkos/pull/8668